packages/lending/src/protocols/aave-v2/lending-protocol.ts

```diff
@@ -297,11 +299,17 @@ export class LendingProtocol extends Protocol {
         usageAsCollateralEnabled = usageAsCollateralEnabled && userBalance.usageAsCollateralEnabled;
       }
 
+      const lstApy = lstTokenAPYMap[token.address.toLowerCase()] || '0';
+      const grossApy =
+        lstApy === '0' ? reserveData.supplyAPY : BigNumberJS(reserveData.supplyAPY).plus(lstApy).toString();
```

Comment:
Since the interface of BIgNumberJS.plus() is plus(n: BigNumber.Value ...), I suggest transforming the `lstApy` from `string` to `BigNumberJS` to avoid any unexpected error (though the calculation is correct in the current implementation). What do you think?

Reply:
https://mikemcl.github.io/bignumber.js/#plus
```
BigNumber.Value
type Value = string | number | Instance;
```
I believe using a `string` as property is fine